services/cooperation.js

```diff
@@ -29,18 +29,29 @@ const cooperationService = {
     })
   },
 
-  updateCooperation: async (id, currentUserRole, updateData) => {
+  updateCooperation: async (id, currentUser, updateData) => {
+    const { id: currentUserId, role: currentUserRole } = currentUser
     const { price, status } = updateData
 
     if (price && status) {
       throw createError(409, VALIDATION_ERROR('You can change only either the status or the price in one operation'))
     }
 
     const cooperation = await Cooperation.findById(id)
+    if (!cooperation) {
+      throw createError(DOCUMENT_NOT_FOUND(Cooperation.modelName))
```

Comment:
model name should be passed inside of array

Reply:
and the same in endpoints below, cause if you pass model name without array it will show "were not found" instead of "was not found"

Reply:
okey, maybe something was wrong on my side when it does not in proper way